scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -175,16 +177,43 @@ MacRxOk (Ptr<DmgWifiMac> WifiMac, Ptr<OutputStreamWrapper> stream,
     }
 }
 
+void
+CwTrace (uint32_t oldCw, uint32_t newCw)
+{
+  // std::cout << "Old Cw: " << oldCw << ", New Cw: " << newCw << std::endl;
```

Comment:
Instead of commenting the entire method (making it idealy useless), use `NS_LOG_DEBUG` in place of `std::cout`, so it can be activated directly from command line